spec/features/officing/results_spec.rb

```diff
@@ -158,4 +159,7 @@
     within('#total_results') { expect(page).to have_content('66') }
   end
 
+  after do
```

Comment:
Can you move this block right after the `background` block? It's easier to understand what's going to happen that way :smile_cat: